build.rs

```diff
@@ -1,17 +1,32 @@
+#[cfg(feature = "rt")]
+use std::env;
+
+#[cfg(feature = "rt")]
+const DEFAULT_HEAP_SIZE: usize = 16 * 1024;
+#[cfg(feature = "rt")]
+const DEFAULT_STACK_SIZE: usize = 16 * 1024;
+
 #[cfg(feature = "rt")]
 fn main() {
     use imxrt_rt::{Family, FlexRamBanks, Memory, RuntimeBuilder};
 
+    let heap_size = env::var("TEENSY4_HEAP_SIZE")
+        .map(|val| val.parse::<usize>().unwrap_or(DEFAULT_HEAP_SIZE))
```

Comment:
Since this is `build.rs`, should we fail on an unparsable value? The feels like the kind of error that should fail the build.

Reply:
I could see it either way.  I definitely want empty to be the default of 16K though.

Reply:
Yes, empty should be the default, but if someone fat fingers `1o24`. that should probably fail.